docs/sources/write/_index.md

```diff
@@ -1,29 +1,31 @@
 ---
-title: Write documentation
-menuTitle: Write
-description: Learn about Grafana Labs' technical documentation writing guidelines.
-weight: 400
 aliases:
   - /docs/writers-toolkit/writing-guide/
   - /docs/writers-toolkit/write/
-Keywords:
+date: 2024-02-29
+description: Learn about Grafana Labs' technical documentation writing guidelines.
+keywords:
   - writing
   - guidelines
+menuTitle: Write
+title: Write documentation
+weight: 400
 ---
 
 # Write documentation
 
-This writing guide defines the structured authoring environment we use to create documentation at Grafana Labs.
+This writing guide defines how Grafana Labs writes documentation.
 If you write technical documentation for Grafana Labs, familiarize yourself with these guidelines.
 
-We write our documentation using [U.S. English](https://developers.google.com/style/translation).
+## Before you write
 
-The guidelines are for anyone who is interested in improving Grafana Labs' technical content. They are intended to guide you on your documentation journey, whether you are requesting a change, editing a topic, or writing a set of documentation for a new product or feature from scratch.
+To learn how to structure documentation, refer to [Structure](https://grafana.com/docs/writers-toolkit/structure/).
 
-If you are already familiar with the guidelines, feel free to get started from one of the [templates](https://github.com/grafana/writers-toolkit/tree/main/docs/static/templates).
```

Comment:
I think linking to Structure is preferable over letting the reader assume they know the style without knowing what each topic covers. The `section` shortcode includes descriptions for each topic so that a reader can make a more informed decision about what content to skip.